docs/simplecontroller_config.md

```diff
@@ -0,0 +1,39 @@
+# SimpleControllerConfiguration
+
+## Rate
+
+*float64*
+
+Rate is the idle time coefficient for user actions that will be performed sequentially.
+
+- A rate < 1.0 will run actions at a faster pace.
+- A rate == 1.0 will run actions at the default pace.
+- A rate > 1.0 will run actions at a slower pace.
+
+This value is multiplied with [WaitAfterMs](#WaitAfterMs).
+
+## Actions
+
+*[]ActionDefinition*
+
+Actions are the user action definitions that will be run by the SimpleController.
+
+### ActionDefinition
+
+#### ActionId
+
+*string*
+
+Action name which is mapped to SimpleController's actions. Available actions can be found [here](https://github.com/mattermost/mattermost-load-test-ng/blob/master/loadtest/control/simplecontroller/controller.go#L137).
+
+#### WaitAfterMs
+
+*integer*
+
+Wait time in milliseconds after the action is performed.
+
+#### RunFrequency
+
+*integer*
+
+The value of how often the action will be performed. The higher the value the lesser possibility to run the action.
```

Comment:
This is my fault and we might want to address it before it's too late. Until today it was fine but reading this now really makes me think something is not right.

Should we name this `RunPeriod`? I know it's not ideal but better than frequency for sure. High frequency cannot mean doing something less frequently :man_facepalming:

Reply:
I was wondering the same actually. In fact I double checked the code and tried to figure out why it is named liked this. Eventually it is a frequency but inversely proportional 🙃

Reply:
Yeah .. the concept is "once in so many times". If there is a word for that, we can use it. But I am not aware of anything. Whichever is the closest to the concept works for me :)

Reply:
Yes, the idea behind *Period* was that if we take the loop iteration (cycle) as our unit of time then we can say the period of an action is that value we define there. A period of 10 means it gets run once every 10 cycles/iterations.